.github/bin/github-pages-setup.sh

```diff
@@ -27,7 +29,7 @@ echo "Git hash:    $GIT_HASH"
 echo "Git version: $GIT_VERSION"
 
 PAGES_DIR=/tmp/pages
-rm -rf $PAGES_DIR
+rm -rf $PAGES_DIR || true
```

Comment:
Do we need this  `|| true` ? The `-rf` option should make sure that we get an exit code that is 0 regardless if the directory already exists.

Reply:
Good catch. We don't need it. I added it "just in case" to be extra sure that my build doesn't fail here.

Reply:
I've removed this one.